Packs/CheckPointSandBlast/Integrations/CheckPointSandBlast/CheckPointSandBlast.py

```diff
@@ -0,0 +1,919 @@
+
+    Args:
+        timestamp_string (str, optional): Holds a timestamp to be converted.
+            Defaults to '0'.
+
+    Returns:
+        str: A string with the timestamp in datetime format.
+    """
+    timestamp = int(timestamp_string) * 1000
+    return timestamp_to_datestring(timestamp)
+
+
+def get_quota_context_output(outputs: Dict[str, Any]) -> Dict[str, Any]:
+    """
+    Convert outputs keys to PascalCase and convert any timestamp to date format.
+
+    Args:
+        outputs (Dict[str, Any]): API key quota information.
+
+    Returns:
+        Dict[str, Any]: outputs in a more readable form.
+    """
+    response_by_context = {
+        'RemainQuotaHour': 'remain_quota_hour',
+        'RemainQuotaMonth': 'remain_quota_month',
+        'AssignedQuotaHour': 'assigned_quota_hour',
+        'AssignedQuotaMonth': 'assigned_quota_month',
+        'HourlyQuotaNextReset': 'hourly_quota_next_reset',
+        'MonthlyQuotaNextReset': 'monthly_quota_next_reset',
+        'QuotaId': 'quota_id',
+        'CloudMonthlyQuotaPeriodStart': 'cloud_monthly_quota_period_start',
+        'CloudMonthlyQuotaUsageForThisGw': 'cloud_monthly_quota_usage_for_this_gw',
+        'CloudHourlyQuotaUsageForThisGw': 'cloud_hourly_quota_usage_for_this_gw',
+        'CloudMonthlyQuotaUsageForQuotaId': 'cloud_monthly_quota_usage_for_quota_id',
+        'CloudHourlyQuotaUsageForQuotaId': 'cloud_hourly_quota_usage_for_quota_id',
+        'MonthlyExceededQuota': 'monthly_exceeded_quota',
+        'HourlyExceededQuota': 'hourly_exceeded_quota',
+        'CloudQuotaMaxAllowToExceedPercentage': 'cloud_quota_max_allow_to_exceed_percentage',
+        'PodTimeGmt': 'pod_time_gmt',
+        'QuotaExpiration': 'quota_expiration',
+        'Action': 'action',
+    }
+
+    context_outputs_with_date = [
+        'HourlyQuotaNextReset',
+        'MonthlyQuotaNextReset',
+        'CloudMonthlyQuotaPeriodStart',
+        'PodTimeGmt',
+        'QuotaExpiration',
+    ]
+
+    output: Dict[str, Any] = {}
+
+    for context_output, response in response_by_context.items():
+        output[context_output] = outputs.get(response)
+
+    for key in context_outputs_with_date:
+        output[key] = get_date_string(output[key])
+
+    return output
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    """
+    Getting data from instance setting and setting up the class Client with an API key.
+    Checking user input command with if statements and a dictionary.
+    """
+    params: Dict[str, Any] = demisto.params()
+    args: Dict[str, Any] = demisto.args()
+    command = demisto.command()
+
+    api_key = params['credentials']['password']
+    host = params['url']
+    verify_certificate = not params.get('insecure', False)
+    proxy = params.get('proxy', False)
+
+    commands = {
+        'sandblast-query': query_command,
+        'sandblast-upload': setup_upload_polling_command,
+        'sandblast-download': download_command,
+        'sandblast-quota': quota_command,
```

Comment:
please add the file command

Reply:
Sorry for the misunderstanding about the implementation, working on it now.